.github/workflows/main.yml

```diff
@@ -7,46 +7,62 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ghidra: ["10.0.4"]
+        ghidra: ['10.1']
     steps:
       - uses: actions/checkout@v2
+
       - uses: actions/setup-java@main
         with:
           java-version: '11'
-          java-package: jdk
-          architecture: x64
-          distribution: 'temurin'
+          distribution: temurin
+          
+      - name: Get shortened SHA
+        run: echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-7`" >> $GITHUB_ENV
+          
       - uses: er28-0652/setup-ghidra@master
         with:
           version: ${{ matrix.ghidra }}
+          
       - uses: actions/checkout@v2
         with:
           repository: astrelsky/Ghidra-Cpp-Class-Analyzer
           path: Ghidra-Cpp-Class-Analyzer
+          
       - uses: eskatos/gradle-command-action@v2
         with:
           gradle-version: 6.9
           build-root-directory: Ghidra-Cpp-Class-Analyzer
           arguments: buildExtension -PGHIDRA_INSTALL_DIR=${{ env.GHIDRA_INSTALL_DIR }}
+          
       - name: Install the class analyzer as an extension forcefully
         run: |
            mkdir -p ~/.ghidra/.ghidra_${{ matrix.ghidra }}_PUBLIC/Extensions/
            unzip Ghidra-Cpp-Class-Analyzer/dist/*.zip -d ~/.ghidra/.ghidra_${{ matrix.ghidra }}_PUBLIC/Extensions/
-      - uses: actions/upload-artifact@v2
+      - uses: svenstaro/upload-release-action@1d71c233f746f1dbd928efaf6de63056aa65f6eb
         with:
-          name: Ghidra-Cpp-Class-Analyzer
-          path: Ghidra-Cpp-Class-Analyzer/dist/*.zip
-          if-no-files-found: warn
-      - name: Cleanup
-        run: rm -rf Ghidra-Cpp-Class-Analyzer
+          release_name: Release for ${{ matrix.ghidra }}
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: Ghidra-Cpp-Class-Analyzer/dist/*.zip
+          tag: ${{ env.SHORT_SHA }}
+          overwrite: true
+          file_glob: true
+          
       - uses: eskatos/gradle-command-action@v2
         with:
           gradle-version: 6.9
           arguments: buildExtension -PGHIDRA_INSTALL_DIR=${{ env.GHIDRA_INSTALL_DIR }}
-      - uses: actions/upload-artifact@v2
+          
+      - uses: svenstaro/upload-release-action@1d71c233f746f1dbd928efaf6de63056aa65f6eb
         with:
-          name: GhidraOrbis
-          path: dist/*.zip
-          if-no-files-found: warn
-
+          release_name: Release for ${{ matrix.ghidra }}
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: dist/*.zip
+          tag: ${{ env.SHORT_SHA }}
+          overwrite: true
+          file_glob: true
+          
+      - name: Cleanup
+        run: |
+             rm -rf Ghidra-Cpp-Class-Analyzer
+             rm -rf dist
```

Comment:
Unless this is preserved via cache or something this is unnecessary,
```suggestion
```

Reply:
Cleanup is needed otherwise it will include the other extension stuff 
https://github.com/illusion0001/GhidraOrbis/releases/tag/4f40ff0